R/downloadbuttons.R

```diff
@@ -0,0 +1,541 @@
+## MetaInsight - downloadable files for data uploading instructions - code.
```

Comment:
I can't tell in this file what is existing code and what is old code that you have copied and pasted, so ignore any comments that don't apply to new code. Can you document the new functions to make it easier to pick out what is new?

Reply:
Documented functions